ios/engine/KMEI/KeymanEngine/Classes/InputViewController.swift

```diff
@@ -130,6 +130,9 @@ private class CustomInputView: UIInputView, UIInputViewAudioFeedback {
     }
     let topBarDelta = hideBanner ? 0 : InputViewController.topBarHeight
 
+    // Sets height before the constraints, as it's the height constraint that triggers OSK resizing.
+    keymanWeb.setBannerHeight(to: Int(InputViewController.topBarHeight))
```

Comment:
Does this reduce the banner flicker as well?

Reply:
This does nothing to address keyboard/banner flicker.